### PR TITLE
Improve Laplace Demo Notebook and add minimal README

### PR DESCRIPTION
**Pull request by SauravMaheshkar**
## Description

This PR is related to #312. Changes are minimal such as "beautification" of the Notebook by adding a TOC at the top, making some LaTeX equations bigger to make them more readable, deleting redundant code cells. I've also added a minimal `README.md` within the `examples/laplace_demo/` dir.

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [ ] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [x] My changes are covered by tests

### CODE REVIEW

**Comment by SauravMaheshkar** [comment]
Request for Review @chinmayshah99

File: examples/laplace_demo/laplace.ipynb [hunk 1]
```diff
@@ -1,21 +1,21 @@
 {
```
Comment (chinmayshah99) [inline examples/laplace_demo/laplace.ipynb:1]: <p>I cannot visualize the formulas here. Are they visible?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PyDP/pull/404/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Comment (chinmayshah99) [inline examples/laplace_demo/laplace.ipynb:1]: <p>This language needs work. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PyDP/pull/404/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

**Review by chinmayshah99 — CHANGES_REQUESTED** [review]
Language-specific issues.

File: examples/laplace_demo/laplace.ipynb (hunk 1 above)
Comment (SauravMaheshkar) [reply examples/laplace_demo/laplace.ipynb:1]: Screenshot 2021-12-14 at 5 40 45 PM
Screenshot 2021-12-14 at 5 40 57 PM


I can visualize the formulas on both ReviewNB and Github's rendering of the notebook. Can you please elaborate ?

Comment (SauravMaheshkar) [reply examples/laplace_demo/laplace.ipynb:1]: <p>Uhmm 😅 I haven't actually changed any of this. But sure I'll look into this as well.</p>

**Comment by SauravMaheshkar** [comment]
@chinmayshah99 I've updated the conclusion section in da7990e. LMK if it looks okay.

File: examples/laplace_demo/laplace.ipynb (hunk 1 above)
Comment (chinmayshah99) [reply examples/laplace_demo/laplace.ipynb:1]: > @chinmayshah99 I've updated the conclusion section in [da7990e](https://github.com/OpenMined/PyDP/commit/da7990ef4001d84f4473db9e581814a81b74ab5e). LMK if it looks okay.


Can we keep the existing part of the conclusion too as that is the technical summary?

Comment (SauravMaheshkar) [reply examples/laplace_demo/laplace.ipynb:1]: Uhm sure I'll add those back

Comment (SauravMaheshkar) [reply examples/laplace_demo/laplace.ipynb:1]: @chinmayshah99 how does it look after 64ba683

**Review by chinmayshah99 — APPROVED** [review]

**Review by chinmayshah99 — APPROVED** [review]
LGTM!

**Pull request merged**